### DISCUSSION TITLE
random color using format-tab-title

### DISCUSSION THREAD

**Discussion by swnff** [wezterm/wezterm#4449] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Hyprland

### WezTerm version

wezterm 20231004-113117-11dec45f

### Ask your question!

Hello!

Is it possible to change the color, like this gray to white and this "dark gray" to  black?
![image](https://github.com/wez/wezterm/assets/97562462/f78d229b-541f-4e59-8b89-a16889ac9076)

**Comment by bew** [comment, marked as answer]
Tab colors config is documented here:
https://wezfurlong.org/wezterm/config/appearance.html#retro-tab-bar-appearance
